config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  match "/404", :to => "errors#not_found", :via => :all
+  match "/500", :to => "errors#internal_server_error", :via => :all
+
```

Comment:
Yep that right - the new error pages are created dynamically and we will need to have something on nginx incase the app goes down